articles/virtual-machines/workloads/sap/azure-monitor-sap-quickstart.md

```diff
@@ -80,11 +80,21 @@ Sign in to the Azure portal at https://portal.azure.com
 
 > [!IMPORTANT]
 > To configure OS (Linux) provider, ensure that Node_Exporter is installed in each BareMetal instance. For more information, see [Node_Exporter](https://github.com/prometheus/node_exporter)
+> ALso download the Node Exporter from [this](https://prometheus.io/download/#node_exporter) link only.
 
 2. Input a name, which will be the identifier for the BareMetal Instance.
 3. Input the Node Exporter Endpoint in the form of http://IP:9100/metrics.
 4. When finished, select **Add provider**. Continue to add more providers as needed or select **Review + create** to complete the deployment. 
 
+>[Note]
+If Node Exporter is not able to connect with the OS provider on the BM or VM instance then please check the below settings.
+
+1.	Is the Azure monitor is in the same vnet as that of the host machine ?
+2.	Is firewall enabled on host for traffic on port 9100 ? Can be confirmed by running “curl http:<private-ip>:9100/metrics”  from a baremetal/vm running in the same vnet as the host with node_exporter running.
+a.	Incase its not, you can use below commands to enable it
```

Comment:
The formatting of the sub points is not appearing properly in the preview

Reply:
Fixed the same.